server/src/main/java/io/pravega/schemaregistry/storage/client/TableStore.java

```diff
@@ -178,18 +176,19 @@ protected void doStop() {
     }
 
     public CompletableFuture<List<Version>> updateEntries(String tableName, Map<byte[], VersionedRecord<byte[]>> batch) {
+        List<TableSegmentEntry> entries = batch.entrySet().stream().map(x -> {
+            return x.getValue().getVersion() == null ?
+                    TableSegmentEntry.notExists(x.getKey(), x.getValue().getRecord()) :
+                    TableSegmentEntry.versioned(x.getKey(), x.getValue().getRecord(), x.getValue().getVersion().getVersion());
+        }).collect(Collectors.toList());
         return withRetries(() -> {
```

Comment:
Same as last comment (Block `{...}` is unecessary.

Reply:
same as above. block is for better organization of code